src/trsp/src/GraphDefinition.cpp

```diff
@@ -157,9 +185,10 @@ void GraphDefinition::explore(
     GraphEdgeInfo* new_edge;
     // int new_node;
     double totalCost;
-    for(i = 0; i < vecIndex.size(); i++)
+    //for(i = 0; i < vecIndex.size(); i++)
+    for (const auto &vecindex: vecIndex)    
```

Comment:
`vecIndex` is a vector of index
`vecindex` is an element of the vector so maybe might be better to use `index` (without the `vec`)